src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -704,7 +730,7 @@ service JobManagementService {
     }
 
     /// Equivalent to ObserveJob, applied to all active jobs. This stream never completes.
-    rpc ObserveJobs (google.protobuf.Empty) returns (stream JobChangeNotification) {
+    rpc ObserveJobs (ObserveJobsQuery) returns (stream JobChangeNotification) {
```

Comment:
We should add second method here, not change the existing one, as this would be an incompatible change.

Reply:
grpc and protobufs should guarantee this to be backwards and forwards compatible. I did some testing here, unless you know there is some other incompatibility here?

Reply:
I assumed that replacing `Empty` with `ObserveJobsQuery` will break existing clients. If not, lets make this change.